gatsby/config/gatsby-browser.tsx

```diff
@@ -1,6 +1,25 @@
 import 'assets/ds/scss/front.scss';
 import React from 'react';
 import Layout from '../src/layouts/default-layout';
+import { ApplicationInsights } from '@microsoft/applicationinsights-web';
+
 export const wrapPageElement = ({ element }) => {
   return <Layout>{element}</Layout>;
 };
+
+const appInsights = new ApplicationInsights({
+  config: {
+    instrumentationKey: process.env.GATSBY_APPINSIGHTS_KEY,
+  },
+});
+
+export const onInitialClientRender = () => {
+  appInsights.loadAppInsights();
+  appInsights.trackPageView();
+};
+
+export const onRouteUpdate = () => {
+  appInsights.trackPageView();
+};
+
+(window as any).appInsights = appInsights;
```

Comment:
nevim proc, ale to nevidi `gatsby/src/types/global.d.ts` :(

Reply:
můžeš je zkusit importnout v tsconfigu ručně

Reply:
@breberafilip promin, ale jak?

Reply:
include "./src/types/global.d.ts", ale zkus to co jsem psal v komentu níž

Reply:
> do include nebo types "./src/types/global.d.ts",

ne :(

Reply:
oh fuck, to fuguje :D, diky moc!